src/model_output_summary.R

```diff
@@ -0,0 +1,60 @@
+model_output_summary <- function(target_date){
+  ## A function to write a summary table of `model-output` and `model-metadata`
```

Comment:
should not block us from merging this, but I'd suggest using roxygen-style documentation features here, including for the example at the bottom of the file. No reason not to do this, given that there are standardized ways of structuring everything you've included here.